roxie/ccd/ccdfile.hpp

```diff
@@ -86,11 +86,19 @@ interface ITranslatorSet;
 interface IInMemoryIndexManager;
 interface IResolvedFileCache;
 
+enum class FileFormatMode
```

Comment:
Long term this probably needs to be a string - so that it can cope with extensible file formats.  That can be changed later.